openapi.yml

```diff
@@ -5569,6 +5596,143 @@ components:
                 pagination:
                     $ref: '#/components/schemas/Pagination'
                     description: ''
+        Registration:
+            title: Root Type for Registration
+            description: ''
+            type: object
+            properties:
+                id:
+                    format: int32
+                    description: Unique identifier of `Registration`
+                    type: integer
+                created_at:
+                    description: Timestamp when this `Registration` was created
+                account_id:
+                    description: Unique identifier of the `Account` this `Registration` belongs to
+                    type: integer
+                visitor_id:
+                    description: Unique identifier of the `Visitor` this `Registration` belongs to
+                    type: integer
+                pre_registration_id:
+                    description: Unique identifier of the `PreRegistration` this `Registration` belongs to
+                    type: integer
+                collected_information:
+                    description: A hash with all the different types of possible pages on the `Registration` flow.
```

Comment:
There is no `type` defined for this, which means it can't be used for API documentation, SDK generation, or compiled-language consumption. It needs to be an actual data type with a defined structure.

Reply:
We can't have free-form json blobs anywhere in the SDK, so anywhere you have a "hash" it needs to be turned in to a data type